tests/ImageSharp.Tests/TestUtilities/ImageProviders/ITestImageProvider.cs

```diff
@@ -16,7 +16,9 @@ namespace SixLabors.ImageSharp.Tests
     public interface ITestImageProvider
```

Comment:
The file name is misleading now, we should either always move classes to a separate files, or change the rules.